account_statement_import_txt_xlsx/tests/fixtures/debit_credit_amount.csv

```diff
@@ -1,5 +1,5 @@
 "Date","Label","Debit","Credit","Balance","Partner Name","Bank Account"
-"12/15/2018","Credit 20.00","0.00","20.00","-10.00","John Doe","123456789"
+"12/15/2018","Debit empty, Credit 20",,"20","-10.00","John Doe","123456789"
```

Comment:
```suggestion
"12/15/2018","Debit empty, Credit 20.00",,"20.00","-10.00","John Doe","123456789"
```
🤔

Reply:
I set 20 to test that it works with an integer. Should I write `Debit empty, Credit integer`?

Reply:
I have now written `Debit test empty, Credit test integer` to be clear on why I have empty / integer values.